src/hooks/useViewState.ts

```diff
@@ -137,15 +137,17 @@ export function useViewState({
            * very difficult to interact with. We create an element with maximum focus priority and
            * focus it so that when we delete it the user will be at the start of the focus order
            * just like if they had freshly loaded the page. */
-          const shouldFocus = autofocus !== 'off';
-          if (savedActiveElement !== null && shouldFocus) {
-            savedActiveElement.focus();
-          } else {
-            const tempInteractiveEl = document.createElement('span');
-            tempInteractiveEl.setAttribute('tabindex', '1');
-            document.body.prepend(tempInteractiveEl);
-            tempInteractiveEl.focus();
-            tempInteractiveEl.remove();
+          const shouldAutoFocus = autofocus !== 'off';
+          if (shouldAutoFocus) {
+            if (savedActiveElement !== null) {
+              savedActiveElement.focus();
+            } else {
+              const tempInteractiveEl = document.createElement('span');
+              tempInteractiveEl.setAttribute('tabindex', '1');
+              document.body.prepend(tempInteractiveEl);
```

Comment:
Not every document has `document.body`. This will break if the UI is loaded in an SVG or XML document.

If the goal is to scroll to the top of the document, I think all we should be doing is `document.documentElement.scrollIntoView()` and don't need this temporary element at all

```suggestion
              document.body.prepend(tempInteractiveEl);
```

Reply:
also I was confused about what this code block is supposed to do but after reviewing https://github.com/transcend-io/consent-manager-ui/pull/158 it makes sense to me. I just think we need to swap body with documentElement

Reply:
just swapped body with documentElement!